src/index.css

```diff
@@ -32,7 +32,7 @@ body {
 .logo {
   margin: 0 1rem;
   font-family: 'Roboto', sans-serif;
-  color: yellow;
+  color: orchid;
```

Comment:
- [ ] As the project requirements mentioned your final design should match Zeplin's design, So please fix your styling by adding the correct font type, font size, font weight, and colors for all elements on the page.

![image](https://user-images.githubusercontent.com/68796232/176277419-0d6e78de-6fcc-4752-bd47-2dab6975ad84.png)


- [ ] Kindly fix the width and spaces between all input fields and try to stick as much as possible to the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/176277395-024bcf72-0338-4667-9dfe-7adf1a060dc9.png)  | ![image](https://user-images.githubusercontent.com/68796232/176277369-e325f78d-7d33-40bd-b089-4171306291c3.png) |